test/hashtags.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
Love the tests. Only suggestion is lets use much more descriptive Conveys so we can use these as API docs.

Reply:
Tried to make them a little more descriptive - feel free to suggest specific ones that might need more work :-)